spring-cloud-kubernetes-config/pom.xml

```diff
@@ -61,6 +61,10 @@
 			<groupId>org.springframework.security</groupId>
 			<artifactId>spring-security-rsa</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
I think this should be optional....ie retry is only going to happen if the app has spring-retry on the classpath